firefox-ios/Client/Frontend/Browser/Tabs/Models/InactiveTabsModel.swift

```diff
@@ -4,7 +4,8 @@
 
 import Foundation
 
-struct InactiveTabsModel: Equatable {
+struct InactiveTabsModel: Equatable, Identifiable, Hashable {
+    let id: String
     var tabUUID: TabUUID
```

Comment:
Just a couple thoughts: this new `id` property looks like it's duplicating the `tabUUID`, is it needed? Can they ever be different?

Offhand I believe we can get `Identifiable` conformance just using the existing `tabUUID`, via: `var id: TabUUID { return tabUUID }`

We would want to double-check that `tabUUID` won't change, however; I'm not actually sure why it's a `var` instead of a `let`.

Reply:
Thanks! I was actually thinking about this when I was implementing this and I was curious on your thoughts on this so I'm glad you commented. I'll play around with this struct a bit... seems like these should be let and I'll change the way we're declaring the ids.